idseq_dag/steps/run_star.py

```diff
@@ -116,17 +116,19 @@ def __init__(self, *args, **kwargs):
         self.output_histogram_file = self.additional_attributes.get("output_histogram_file")
         requested_insert_size_metrics_output = bool(self.output_metrics_file or self.output_histogram_file)
 
-        # Check if the STAR genome was generated with a gtf file
+        # Check if the STAR genome was generated with an organism-specific gtf file
         #  This file will be in the same s3 directory, it is needed to prevent overestimation
         #  of insert size for RNA because genomic alignments of RNA may cross introns.
+        #  The ERCC.gtf file is not sufficient for this purpose. An organism-specific gtf
```

Comment:
When is a ERCC.gtf file generated vs other gtf files. Please add to the comment.